BrowserKit/Sources/MenuKit/MenuTableView.swift

```diff
@@ -75,13 +83,18 @@ final class MenuTableView: UIView,
         _ tableView: UITableView,
         heightForHeaderInSection section: Int
     ) -> CGFloat {
+        if isRedesignEnabled {
```

Comment:
nit: This is hard to read/understand, could you please add some documentation explaining the different cases